EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -21,17 +21,30 @@
 
 class AttachRefreshTokenOnSuccessListener
 {
-    protected $userRefreshTokenManager;
+    protected $refreshTokenManager;
     protected $ttl;
     protected $validator;
     protected $requestStack;
+    protected $getUsername;
+    protected $userManager;
 
-    public function __construct(RefreshTokenManagerInterface $refreshTokenManager, $ttl, ValidatorInterface $validator, RequestStack $requestStack)
+    public function __construct(
+        RefreshTokenManagerInterface $refreshTokenManager,
+        int $ttl,
+        ValidatorInterface $validator,
+        RequestStack $requestStack,
+        string $getUsername,
+        UserManager $userManager
```

Comment:
This is a hard dependency on FOSUserBundle but you should consider that not all uses it